flyway/src/main/scala/com/avast/sst/flyway/FlywayConfig.scala

```diff
@@ -0,0 +1,25 @@
+package com.avast.sst.flyway
+
+import java.nio.charset.{Charset, StandardCharsets}
+
+import org.flywaydb.core.api.MigrationVersion
+
+final case class FlywayConfig(baselineOnMigrate: Boolean = false,
+                              baselineVersion: Option[MigrationVersion] = None,
```

Comment:
sice we have `baselineVersion: Option[MigrationVersion] = None`, maybe we should also have something like
```scala
target: Option[MigrationVersion] = None
...
.setTarget(target)
```

Reply:
ok